### ISSUE TITLE
Update the UI

### ISSUE THREAD

**Issue by Bashamega** [scratchblocks/scratchblocks#523]
Hello:)
I have noticed that the UI is quite old, and it doesn't have a good color contrast.
I can turn this project into a great website with a decent scratch themes website.
